services/slam/builtin/cartographer_int_test.go

```diff
@@ -55,7 +61,17 @@ func testCartographerPositionAndMap(t *testing.T, svc slam.Service) {
 	test.That(t, actualMIME, test.ShouldResemble, "pointcloud/pcd")
 	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
 
-	// TODO DATA-701 test GetPosition
+	position, err := svc.Position(context.Background(), "test", map[string]interface{}{})
```

Comment:
[nit] can we reorder this so that is matches orbslam_int_test (i.e. position check then map check)